src/metadata/database.rs

```diff
@@ -48,16 +48,16 @@ pub struct Database {
 impl Database {
 	/// Load a database from the given file.
 	pub fn load<P: AsRef<Path>>(path: P) -> Result<Self, error::LoadMetadata> {
-		Database::from(loader::load(BufReader::new(File::open(path)?))?)
+		Database::from(loader::load(BufReader::new(File::open(path)?))?, false)
 	}
 
 	/// Parse a database from the given string.
 	pub fn parse<S: AsRef<str>>(content: S) -> Result<Self, error::LoadMetadata> {
-		Database::from(loader::load(Cursor::new(content.as_ref()))?)
+		Database::from(loader::load(Cursor::new(content.as_ref()))?, false)
 	}
 
 	/// Create a database from a loaded database.
-	pub fn from(meta: Vec<loader::Metadata>) -> Result<Self, error::LoadMetadata> {
+	pub fn from(meta: Vec<loader::Metadata>, check_regex: bool) -> Result<Self, error::LoadMetadata> {
```

Comment:
This is a braking change.

Furthermore the api between `from`, `load` and `parse` is no inconsistent.

If we anyway do a braking change we probably want to have some  *opaque*  `Options` type we can pass into all of `from`, `load` and `parse`.